internal/sync.go

```diff
@@ -296,3 +307,34 @@ func (s *syncGSuite) ignoreGroup(name string) bool {
 
 	return false
 }
+
+func (s *syncGSuite) allowGroup(name string) bool {
+	if len(s.cfg.AllowGroups) == 0 {
+		return true
+	}
+
+	for _, g := range s.cfg.AllowGroups {
+		if strings.HasPrefix(name, g) {
+			return true
+		}
+	}
+
+	return false
+}
+
+func (s *syncGSuite) allowPattern(name string) bool {
+	if len(s.cfg.AllowGroups) == 0 {
+		return true
+	}
+	for _, p := range s.cfg.AllowPattern {
+		if p == "" {
```

Comment:
Would accepting a pattern of `*` be less accident prone than simply a blank pattern? Seems too easy for an errant comma causing everyone to match.